README.md

```diff
@@ -5,13 +5,15 @@
 The steps to clone and the project locally is :
 git clone git@github.com:OsvaldoBC/portfolio-setup.git
 
+# usage
+
+ Please use this link to see this project https://osvaldobc.github.io/osvaldojose.github.io/
+
```

Comment:
- [ ] [Optional] It would be better professionally if The deployment link were under `Deployment` or `Live Demo` section instead of `usage` section.
[Hint] You can do it like that:
```
## Live Demo
My portfolio live demo [link](https://osvaldobc.github.io/osvaldojose.github.io/). 
```
and It will look like that:
## Live Demo
My portfolio live demo [link](https://osvaldobc.github.io/osvaldojose.github.io/).